base/system/services/rpcserver.c

```diff
@@ -1672,6 +1609,81 @@ ScmIsValidServiceState(DWORD dwCurrentState)
     }
 }
 
+static
+DWORD
+WINAPI
+ScmStopThread(PVOID pParam)
+{
+    PSERVICE lpService = (PSERVICE)pParam;
+    WCHAR szLogBuffer[80];
+    LPCWSTR lpLogStrings[2];
+
+    /* Check if we are about to stop this service */
+    if (lpService->lpImage->dwImageRunCount == 1)
+    {
+        /* Stop the dispatcher thread. */
+        /* We must not send a control message while hodling the database lock or it can cause timeouts */
+        /* We are sure that the service won't be deleted in the meantime because we still have a reference to it */
```

Comment:
```suggestion
        /* Stop the dispatcher thread.
         * We must not send a control message while holding the database lock, otherwise it can cause timeouts.
         * We are sure that the service won't be deleted in the meantime because we still have a reference to it. */
```

Reply:
Changed. Thanks.